integration_tests/models/schema.yml

```diff
@@ -301,6 +301,25 @@ models:
       - elementary.schema_changes:
           tags: ["schema_changes"]
 
+  - name: users_per_day_weekly_seasonal
+    description: >
+      Volume anomalies on the training should fail when seasonality is Off and pass when seasonality is On. 
+      When Validation data is added, should fail when seasonality is On and pass when seasonality is Off.
+    tests:
+      - elementary.volume_anomalies:
+          alias: volume_anomalies_no_seasonality_anomalies
+          timestamp_column: "updated_at"
+          tags: ["seasonality_volume"]
+          sensitivity: 2
+          backfill_days: 14
```

Comment:
Why 14 backfill days? 
Also how would we have enough data points to calculate the anomaly? 
Won't we have only 2?

Reply:
`backfill_days` means we predict on 14 days.
I'm running it with 210 `days_back` so we have 30 days back on each day of the week.